packages/web-components/src/components/va-need-help/va-need-help.tsx

```diff
@@ -0,0 +1,28 @@
+import { Component, Host, h } from '@stencil/core';
+
+/**
+ * @componentName Need Help?
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+
+
+@Component({
+  tag: 'va-need-help',
+  styleUrl: 'va-need-help.css',
+  shadow: true,
+})
+export class VaNeedHelp {
+
+  render() {
+    return (
+      <Host>
+        <div class="need-help">
+          <h2 id="need-help">Need help?</h2>
```

Comment:
Would this header ever need to be something other than an `h2` (for page accessibility)?

Reply:
No. It appears at the bottom of the page and is currently always an h2. I believe we want to always be sure this is at that level so it can be easily navigated to via assistive tech.